### ISSUE TITLE
Hi, I want to know what does notepad2-mod doing with network?

### ISSUE THREAD

**Issue by SeaHOH** [XhmikosR/notepad2-mod#176]
I was uesed notepad2-mod for many years, but I found it ues my network to do something today, and I don't know what does it doing, is there anybody can help me?
![n2mnet](https://cloud.githubusercontent.com/assets/8470820/19997386/2d774350-a2a1-11e6-9eb4-2399fdd4e65f.png)

**Comment by JonasNo** [comment]
Just a guess. Do you have a network drive or a share or similar connected or explorer plugins that access internet like perhaps dropbox? There's no network code in notepad2-mod. When you use the open/save dialog it opens an explorer dialog, part of windows, standard. That may cause this if you have such connected.

Have had a few notepad2-mod processes running for days. I see no such network activity. Also my firewall would have screamed out loud if it even had tried to.

**Comment by SeaHOH** [comment]
thanks.

I have missing something in description. The firewall has never reminded me anything too, I found this by chance. The firewall is fail, that's strange. And the connection type is not TCP, I can't catch it.

I have tried several times, some notepad2-mod processes has no network active for ever, and others was shown network active just open a file. The latter case, sometimes network active increased with time, sometimes network active just stoped.

There is no plugins like you said in my system. Is anyone happen this like me.
